quantum/dynamic_keymap.c

```diff
@@ -249,6 +254,10 @@ void dynamic_keymap_macro_send(uint8_t id) {
                 break;
             }
         }
+#if DYNAMIC_KEYMAP_MACRO_DELAY > 0
+        send_string_with_delay(data, DYNAMIC_KEYMAP_MACRO_DELAY);
+#else
         send_string(data);
+#endif
```

Comment:
```suggestion
        send_string_with_delay(data, DYNAMIC_KEYMAP_MACRO_DELAY);
```
`send_string(str)` simply calls `send_string_with_delay(str, 0)`

Reply:
> `send_string(str)` simply calls `send_string_with_delay(str, 0)`

Yes . I khow that .
But I think that When DYNAMIC_KEYMAP_MACRO_DELAY equal zero, using `send_string(str)` is a few bytes less than using `send_string_with_delay(str, 0)` .
(It it very important for Pro Micro AVR firmware size)

Reply:
No, it doesn't, because it'll just be running `send_string_with_delay(str, 0)` anyway. The `#if` is completely pointless.

Reply:
Thank you .